analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -797,6 +795,81 @@ def add_arguments_to_parser(parser):
         func=main, func_process_config_file=cmd_config.process_config_file)
 
 
+def validate_analyzer_parameter(analyzer_conf):
```

Comment:
I usually prefer boolean function names to start with `is_`, `was_`, `should_` etc. to imply that its a boolean function. Also, we're specifically checking analyzer configs, not any random parameter to the analyzer.
```suggestion
def is_analyzer_config_valid(analyzer_conf: List[AnalyzerConfig]) -> bool:
```